app/models/question.rb

```diff
@@ -0,0 +1,8 @@
+class Question < ApplicationRecord
+  enum type: {"single choice": 0, "text_question": 1}
```

Comment:
```suggestion
  enum type: {single_choice: 0, text_question: 1}
```
Em đổi sang như thế này nhé để còn áp dụng i18n